src/pages/home/report/ReportAttachments.tsx

```diff
@@ -26,10 +27,10 @@ function ReportAttachments({route}: ReportAttachmentsProps) {
 
     const onCarouselAttachmentChange = useCallback(
         (attachment: Attachment) => {
-            const routeToNavigate = ROUTES.ATTACHMENTS.getRoute(reportID, type, String(attachment.source), Number(accountID));
+            const routeToNavigate = ROUTES.ATTACHMENTS.getRoute(reportID, type, String(attachment.source), Number(accountID), !!isAuthTokenRequired, imageHrefLink);
```

Comment:
Could you please explain why we need to pass `!!isAuthTokenRequired` in this case?

Reply:
I've updated the parameters. Can you check them again?